### PR TITLE
chatbot parameters

### PR DESCRIPTION
**Pull request by navyad**
Issue: [https://github.com/gunthercox/ChatterBot/issues/287](https://github.com/gunthercox/ChatterBot/issues/287)

### CODE REVIEW

**Comment by gunthercox** [comment]
@navyad the changes look good. I'll have to test the changes out but the code looks like it works. (I believe the remaining failing tests are just due to the old parameter styles being passed in)

**Comment by navyad** [comment]
@gunthercox Ok, let me know if you need any help from side.

File: chatterbot/chatterbot.py [hunk 1]
```diff
@@ -14,21 +14,22 @@ def __init__(self, name, **kwargs):
...
+        default_input = {"input_class": 'chatterbot.adapters.input.VariableInputTypeAdapter'}
+        input_kwargs = kwargs.get('input_adapter', default_input)
+        input_adapter = input_kwargs.get('input_class')
 
-        output_adapter = kwargs.get('output_adapter',
-            'chatterbot.adapters.output.OutputFormatAdapter'
-        )
+        default_output = {"output_class": 'chatterbot.adapters.output.OutputFormatAdapter'}
+        output_kwargs = kwargs.get('output_adapter', default_output)
+        output_adapter = output_kwargs.get('output_class')
```
Comment (gunthercox) [inline chatterbot/chatterbot.py]: What are your thoughts on changing `output_class`, `input_class`, `storage_class` and `logic_class` all just to `adapter_class` to keep the parameters consistent across all adapters?

Comment (navyad) [inline chatterbot/chatterbot.py]: @gunthercox  Yeah ... it will be good.

Comment (navyad) [inline chatterbot/chatterbot.py]: @gunthercox that been done.

**Comment by gunthercox** [comment]
@navyad Thank you for the attempt on this. I'm going to have to close this off because tests never passed and there has been a number of changes to the master branch since this pull request was opened. I may reattempt this in the future, but I think I need to redesign it so that it doesn't completely break backwards comparability with code written for previous versions of ChatterBot. Thank you again for your time, I'm sorry this can't get added in right now.

**Comment by navyad** [comment]
@gunthercox Ok.

**Pull request closed without merging**